API/API.mli

```diff
@@ -1714,7 +1714,7 @@ end
 module Locus :
 sig
   type 'a occurrences_gen =
-  | AllOccurrences
+  | AllOccurrences of bool
```

Comment:
It seems like it would be better to have two separate constructors rather than a single one with a boolean argument. That would minimize code changes, that would be much more readable code-wise, and that would be a bit better performance-wise.